<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addReteta</class>
 <widget class="QDialog" name="addReteta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(140, 140, 140);</string>
  </property>
  <widget class="QDialogButtonBox" name="SendRetetaButton">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>630</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="numeReteta">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Numele retetei:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ingrediente">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ingrediente:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Gramaje">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Gramaje (gr): </string>
   </property>
  </widget>
  <widget class="QLabel" name="adaugaReteta">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 italic 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Descriere: </string>
   </property>
  </widget>
  <widget class="QLabel" name="Imagine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Imagine </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addNume">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>39</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="addIngrediente">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="addGramaje">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="addDescriere">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Incarca imagine</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SendRetetaButton</sender>
   <signal>accepted()</signal>
   <receiver>addReteta</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SendRetetaButton</sender>
   <signal>rejected()</signal>
   <receiver>addReteta</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
